package.json

```diff
@@ -47,8 +44,14 @@
     "webmaker-app-icons": "git://github.com/k88hudson/ionicons#teach",
     "webpack": "^1.6.0"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "mocha-phantomjs": "^3.5.3",
+    "phantomjs": "^1.9.16",
+    "simplecrawler": "^0.5.2",
+    "supertest": "^1.0.1"
+  },
```

Comment:
Putting packages in `devDependencies` is a lot more useful now since Heroku won't install them when deploying. This is just a minimal set of them--we can probably move more packages into this category later, but for now I'm being really conservative because I don't want to break anything.